### DISCUSSION TITLE
Add navigation to sidebar

### DISCUSSION THREAD

**Discussion by sergeyshevch** [wsvincent/awesome-django#139] [Ideas]
I guess it gan be good idea to add navigation throught the sections to sidebar on https://awesomedjango.org/

**Comment by jefftriplett** [comment]
This is a great idea. I looked into it when I switched the theme out a few weeks ago, and I couldn't see an obvious way to do it unless we broke out each category into its own page, which seemed like it'd be harder to update. 

If anyone knows what I was doing wrong, I'd love to figure it out.

**Comment by jefftriplett** [comment]
I added this a month or two ago and forgot to comment here. Thanks again for the suggestion. I'd like to fill it in more at some point but that might duplicate the TOC. Something to work on going forward though too.
